### PR TITLE
extend selenium-grid scaler

### PR DESCRIPTION
**Pull request by rubroboletus**
Document changes in selenium-grid scaler

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #

### CODE REVIEW

**Comment by tomkerkhove** [comment]
@rubroboletus Mind mentioning feature link in description so I have full context please?

**Comment by rubroboletus** [comment]
> Mind mentioning feature link in description so I have full context please?

It documents PR - https://github.com/kedacore/keda/pull/6055

File: content/docs/2.16/scalers/selenium-grid-scaler.md [hunk 1]
```diff
@@ -105,6 +108,29 @@ spec:
         sessionBrowserName: 'msedge'
 ```
 
+If your selenium browser nodes are not exactly same and you wanna to scale them based on real `slots` settings, set `setSessionsFromHub` to `true` and `setSessionsFromHub` to real number of slots, if you wanna scale from 0.
```
Comment (JorTurFer) [inline content/docs/2.16/scalers/selenium-grid-scaler.md:111]: Is this a typo?
```suggestion
If your selenium browser nodes are not exactly same and you wanna to scale them based on real `slots` settings, set `setSessionsFromHub` to `true` and `sessionsPerNode` to real number of slots, if you wanna scale from 0.
```

**Comment by JorTurFer** [comment]
Could you fix DCO check too?

**Review by JorTurFer — APPROVED** [review]

**Pull request merged**